<h1>Directly Identify RNA-RBP Interactions in Genome-wide Scale</h1>

<p>HITS-CLIP</p>

<hr>

<h1>HITS-CLIP (or CLIP-seq)</h1>

<p><strong>HIT</strong>hroughput <strong>S</strong>equencing of RNAs isolated by <strong>C</strong>rosslinking <strong>I</strong>mmuno<strong>P</strong>recipitation</p>

<hr>

<p>Boring Definition</p>

<hr>

<p>Well, for short</p>

<hr>

<h1>CLIP-seq = Chip-seq</h1>

<p>but, RNA version :P</p>

<hr>

<h1>CLIP-seq = CLIP + Sequencing</h1>

<hr>

<h2>Standard CLIP-seq Pipeline</h2>

<p><img src="images/CLIP-seq.png" ></p>

<p>UV-Crosslink -> IP -> Purify -> Sequencing</p>

<hr>

<h1>Why CLIP?</h1>

<hr>

<h2>Identify candidate target RNA</h2>

<ul>
<li><p>RIP-Chip, indirect or non-phisiological interactions</p></li>
<li><p>UV-Crosslink and IP, Direct interactions</p></li>
</ul>

<hr>

<h1>What Will CLIP-seq Data Offer? </h1>

<hr>

<h1>Precise Binding Position</h1>

<p>Besides candidate target RNA motif</p>

<p><img src="images/candidate-motif.jpg"></p>

<p>Robert B. Darnell, etc. CLIP identifies Nova-Regulated RNA networks in the brain. Science. 2003. </p>

<hr>

<h1>Potentially recuited to machinery</h1>

<p><img src="images/potentially_recuited_to_machinery.jpg"></p>

<p>Tag detected + Significant difference between Ctrl/Knock-out: </p>

<p>It is indicated that NOVA is recuited to influence splicing. </p>

<p>Robert B. Darnell, etc. CLIP identifies Nova-Regulated RNA networks in the brain. Science. 2003. </p>

<hr>

<h1>CLIP Update</h1>

<ul>
<li><p>iCLIP</p></li>
<li><p>PAR-CLIP</p></li>
</ul>

<hr>

<h1>individual-nucleotide resolution CLIP</h1>

<hr>

<p><video width="600" controls="controls">
    <source src="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3169244/bin/jove-50-2638.mov" type="video/mp4">
    Your Browser Does not support Html5, please update your browser. 
</video></p>

<p>iCLIP - Transcriptome-wide Mapping of Protein-RNA Interactions with Individual Nucleotide Resolution. J Vis Exp. 2011. </p>

<hr>

<p><img src="images/iCLIP_pipeline.jpg"></p>

<hr>

<h2>Classic CLIP:</h2>

<ul>
<li>cDNAs truncate prematurely at the crosslinked nucleotide</li>
<li>Truncated cDNAs are lost during the standard CLIP library preparation protocol</li>
</ul>

<h2>iCLIP:</h2>

<p>More efficient intramolecular cDNA circularization </p>

<hr>

<h1>Photoactivatable-Ribonucleoside-Enhanced Crosslinking and Immunoprecipitation</h1>

<hr>

<p><video width="800" controls="controls">
    <source src="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3156069/bin/jove-41-2034.mp4" type="video/mp4">
    Your Browser Does not support Html5, please update your browser.
</video></p>

<p>PAR-CliP - A Method to Identify Transcriptome-wide the Binding Sites of RNA Binding Proteins, J Vis Exp. 2010.</p>

<hr>

<h2>Classic CLIP:</h2>

<ul>
<li>Low efficiency of UV 254 nm RNA-protein crosslinking</li>
<li>Difficult to separate UV-crosslinked target RNA segments from background non-crosslinked RNA fragments also present in the sample</li>
</ul>

<h2>PAR-CLIP:</h2>

<p>Rely on 4-thiouridine (4-SU) and 6-thioguanosine (6-SG)</p>

<ul>
<li>UV 365 nm induces efficient crosslinking of photoreactive nucleoside-labeled cellular RNAs to interacting RBPs</li>
<li>4-SU: T->C transitions</li>
<li>6-SG: G->A mutations. The presense of the changes can be assessed during bioinformatic analysis, and make it possible to seperate them from the background of sequences derived from abundant cellular RNAs.</li>
</ul>

<hr>

<h1>Summary</h1>

<ul>
<li>HITS-CLIP or CLIP-seq</li>
<li>Exact binding position</li>
<li>Direct interaction, candiate target RNA</li>
<li>iCLIP and PAR-CLIP</li>
</ul>

<hr>

<p><img src="images/logo-bonnie-basepairing.jpg"></p>

	<p>Check source code at: <a href="https://github.com/Puriney/clipseq">https://github.com/Puriney/clipseq</a></p>
